vagrant/.gitignore

```diff
@@ -1 +1,3 @@
-.vagrant
\ No newline at end of file
+.vagrant
+*.box
+Berksfile.lock
```

Comment:
thanks -- makes sense. i dont think cookbook's `.gitignore` will apply to this folder since it's a sibling, but we can definitely crib off of it